knowledge-base/security-vulnerability-wrd-r3-2020.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Security vulnerability present when using Web Report Designer
+description: A security vulnerability is introduced with version R3 2020
+type: how-to
+page_title: Security Vulnerability in R3 2020
+slug: security-vulnerability-wrd-r3-2020
+position: 
+tags: security,wrd,issue
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+	<tr>
+		<td>Project Type</td>
+		<td>Web Application</td>
+	</tr>
+	<tr>
+		<td>Report Dessgner</td>
+		<td>Web-based Report Designer</td>
+	</tr>
+</table>
+
+
+## Description
+A security vulnerability is introduced with version R3 2020 (14.2.20.916) and is present in web applications embedding the [Web-based Report Designer](https://docs.telerik.com/reporting/web-report-designer).
+
+## Solution
+We strongly recommend upgrading to version R3 2021 SP2 (15.2.21.1125) or a subsequent one of Progress Telerik Reporting. This version mitigates the issue completely. In case upgrading is not applicable to your scenario, please contact the support service team for detailed instructions how to mitigate the issue. 
```

Comment:
Just a thing about the wording here: AFAIK "mitigate" has meaning of "suppress", "reduce" or "lessen" a certain thing to occur. The solution in our SP actually **resolves** the issue. Of course, in case the users cannot upgrade to SP2, they can **mitigate** it with other means that we could suggest to them.

Reply:
I agree with Vankata

Reply:
Yep. Agree. Changing